frontend/components/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,56 @@
+import classnames from "classnames";
+import React, { ReactChild } from "react";
+
+import ExternalURLIcon from "../../../assets/images/icon-external-url-black-12x12@2x.png";
+
+interface IMainContentProps {
+  children: ReactChild;
+  /** An optional classname to pass to the main content component.
+   * This can be used to apply styles directly onto the main content div
+   */
+  className?: string;
+}
+
+const messageClassName = "sandbox-expiry-message";
+
+const SandboxExpiryMessage = (): JSX.Element => {
+  return (
+    <div className={messageClassName}>
+      <p>Your Fleet Sandbox Expires in about 20 hours.</p>
+      <a
+        href="https://fleetdm.com/docs/deploying"
+        target="_blank"
+        rel="noreferrer"
+      >
+        Learn how to renew or downgrade
+        <img
+          alt="Open external link"
+          className="icon-external"
+          src={ExternalURLIcon}
+        />
+      </a>
+    </div>
+  );
+};
```

Comment:
Later on we'll be adding in the "premium license expiring" message. Does it make sense to import the prompt so that the primary layout component doesn't bloat over time? This works fine for now, just something to think about as we add more prompt messages.

Reply:
I have some ideas for how to handle premium licensing rendering too. Just didn't get around to implementing it for this. It is still a notification for now.